test/CMakeLists.txt

```diff
@@ -59,6 +60,7 @@ list( APPEND test_output
   testoutput/ioda.NC001007.2020031012.nc
   testoutput/viirs_aod.nc
   testoutput/tropomi_no2.nc
+  testoutput/2020100106_metars_small.nc
```

Comment:
This file is missing from the ioda-converters test/testoutput directory.

Reply:
Steve, is it typical to commit the output of the process to the output directory?  I was expecting that running the ctest creates the output file and therefore didn't include it specifically in the git add/commit.  I will pop it in there (in addition to your other suggestions, thanks!)

Reply:
In this repo, the test/testoutput directory is where the expected outputs go and the test/testrun directory is where the outputs from running the tests go.